src/app/shared/components/filter/filter.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="row">
+    <div class="col-9 d-flex flex-wrap">
+        <ul class="list-inline m-0">
+            <li class="p-2 list-inline-item" *ngFor="let item of filterOptions">
+                <button class="btn btn-light" (click)="removeItem(item)"><span class="pr-3">{{item}}</span>X</button>
+            </li>
+        </ul>
+        <div class="p-2">
+            <button *ngIf="filterOptions?.length" class="btn btn-link normal-size-btn p-2" (click)="removeAll()">Clear all</button>
+        </div>
+    </div>
+    <div class="col-3 d-flex justify-content-end">
+        <button class="primary-btn normal-size-btn" (click)="openModal(content)">Filter</button>
+    </div>
+</div>
+<ng-template #content let-c="close" let-d="dismiss">
+    <div class="modal-header">
+        <h4>Filter</h4>
+        <div>X</div>
```

Comment:
Replace this with the icon in the below link
https://material.io/resources/icons/?search=close&icon=close&style=outline
You can use this website for any icons that we may need.